arviz/plots/distcomparisonplot.py

```diff
@@ -60,10 +60,10 @@ def plot_dist_comparison(
         Function to transform data (defaults to None i.e. the identity function)
     legend : bool
         Add legend to figure. By default True.
-    labeller : labeller instance, optional
+    Labeller : Labeller instance, optional
         Class providing the method ``make_pp_label`` to generate the labels in the plot.
         Read the :ref:`label_guide` for more details and usage examples.
-    ax: axes, optional
+    ax : axes, optional
```

Comment:
```suggestion
    ax : (nvars, 3) array-like of matplotlib_axes, optional
```

Here it should only be matplotlib_axes because this plot doesn't work with bokeh.